templates/webapps/galaxy/workflow/run.mako

```diff
@@ -576,15 +576,14 @@ if wf_parms:
 
 %if has_upgrade_messages:
 <div class="warningmessage">
-    Problems were encountered when loading this workflow, likely due to tool
-    version changes. Missing parameter values have been replaced with default.
-    Please review the parameter values below.
+    Warning: Some tools in this workflow have been updated. The workflow may still work, but any new inputs will have default values.
+    Please review the parameter values below to make a decision about whether the changes are reasonable for your experiemnt.
```

Comment:
How about `Some tools [or tool versions] in this workflow have changed` ? 
when you transfer workflows between instances, you can also experience tool downgrades.